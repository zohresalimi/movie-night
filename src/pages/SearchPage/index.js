import React from "react";
import { Grid, Container } from "semantic-ui-react";
import SearchInpput from "../../components/SearchInput";
import MovieList from "../../components/MovieList";

export default function SearchPage() {
  return (
    <Grid id="home" padded>
      <Grid.Row>
        <SearchInpput />
      </Grid.Row>
      <Grid.Row>
        <MovieList />
      </Grid.Row>
    </Grid>
  );
}
