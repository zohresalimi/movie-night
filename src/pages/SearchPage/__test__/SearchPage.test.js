import * as React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";

import SearchPage from "../";

jest.mock("../../../components/SearchInput", () => () => (
  <div>SearchInpput Component</div>
));

jest.mock("../../../components/MovieList", () => () => (
  <div>MovieList Component</div>
));

describe("Search Page Component Testing", () => {
  test("loads items", async () => {
    const { container } = await render(<SearchPage />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
