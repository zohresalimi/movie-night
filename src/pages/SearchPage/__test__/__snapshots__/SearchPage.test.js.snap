// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Page Component Testing loads items 1`] = `
<div
  class="ui stackable grid"
  id="home"
>
  <div
    class="row"
  >
    <div>
      SearchInpput Component
    </div>
  </div>
  <div
    class="row"
  >
    <div>
      MovieList Component
    </div>
  </div>
</div>
`;
